npm/design-system/src/index.ts

```diff
@@ -2,6 +2,8 @@ export * from './components/Button'
 
 export * from './components/CypressLogo/CypressLogo'
 
+export * from './components/FileExplorer/FileExplorer'
```

Comment:
I think it's idiomatic to add a `index.ts` that exports everything. `export * from './FileExplorer`. Then this becomes `export * from './components/FileExplorer'`

Reply:
This was the pattern that @agg23 followed, so I did the same. I don't care one way or another. I've also seen the index.ts pattern and prefer it for convenience, but find it difficult when trying to search for things.

Reply:
I don't care right now. I imagine I will be changing around exports significantly when there starts to be atoms vs final components, so whatever is fine by me.